core/invoker/src/main/scala/whisk/core/container/WhiskContainer.scala

```diff
@@ -92,17 +96,53 @@ class WhiskContainer(
     }
 
     /**
+     * Starts the tracing.
+     */
+    private def startSpan(msg: ActivationMessage): Option[Span] = {
+        val tracerOption = if (GlobalTracer.isRegistered) Option.empty else Some(TracerResolver.resolveTracer)
+        tracerOption foreach GlobalTracer.register
```

Comment:
right, I didn't realize the `TracerResolver.resolveTracer()` can return also null and when using `Some(null)` it actually creates the `Some(null)` what I need here is the `Option(null)` that results in `None`